### DISCUSSION TITLE
Boostrap 4

### DISCUSSION THREAD

**Discussion by gul-ahmad** [cretueusebiu/vform#129] [Q&A]
Thanks for this nice library.
I have a question here
**import { Button, HasError, AlertError } from 'vform/src/components/bootstrap5'**
what if we are using boostrap 4,the old will not work form importing the global components I mean in this way
import { Button, HasError, AlertError } from 'vform';

**Comment by cretueusebiu** [comment]
For Bootstrap 4 you have to use:

`import { Button, HasError, AlertError } from 'vform/src/components/bootstrap4`

This is because now there are 3 variants for the components (Bootstrap 4 & 5 and Tailwind) with support for Vue 2 & 3.
